sky/execution.py

```diff
@@ -383,14 +383,20 @@ def spot_launch(
     # Check the file mounts in the task.
     # Disallow all local file mounts (copy mounts).
     if task.workdir is not None:
-        raise exceptions.NotSupportedError(
-            'Workdir is not allowed for managed spot jobs.')
+        with ux_utils.print_exception_no_traceback():
+            raise exceptions.NotSupportedError(
+                'Workdir is currently not allowed for managed spot jobs.\n'
+                'Hint: use Storage to auto-upload the workdir to a cloud '
+                'bucket.')
     copy_mounts = task.get_local_to_remote_file_mounts()
     if copy_mounts:
-        copy_mounts_str = '\n\t'.join(': '.join(m) for m in copy_mounts)
-        raise exceptions.NotSupportedError(
-            'Local file mounts are not allowed for managed spot jobs, '
-            f'but following are found: {copy_mounts_str}')
+        local_sources = '\t' + '\n\t'.join(
```

Comment:
Should this be `'\n' + '\n\t'.join(...)` to make it easier to see?

Reply:
It looks like this now:
```
» sky spot launch test.yaml                                                                   130 ↵
Task from YAML spec: test.yaml
Launching a new spot task 'sky-a224-zongheng'. Proceed? [Y/n]:
I 08-12 13:39:17 execution.py:364] Field use_spot not specified; defaulting to True.
I 08-12 13:39:17 execution.py:367] No spot recovery strategy specified; defaulting to FAILOVER.
sky.exceptions.NotSupportedError: Local file mounts are currently not allowed for managed spot jobs.
Found local source paths:
        ~/.netrc
        ~/test.yaml
Hint: use Storage to auto-upload local files to a cloud bucket.
```

Reply:
Ahh, that makes sense! Thanks!